features/step_definitions/laundry_room_steps.rb

```diff
@@ -35,4 +35,16 @@
 When(/^I try to access booking link for "([^"]*)"$/) do |date|
   date = Time.parse(date).to_formatted_s(:short)
   visit laundry_room_create_booking_path(@laundry, time: date)
+end
+
+Given("there is a booking from for {string}") do |date|
+  slot = Time.zone.parse(date)
+  slot_to_book = @laundry.schedule
+                     .occurrences(1.week.from_now)
+                     .detect { |occ| occ.beginning_of_hour.localtime == slot.localtime }
+  @user.book! @laundry, time: slot_to_book, amount: 1
+end
```

Comment:
I don't understand. There should be a step like this in our step definitions. I can't see any reason to create a new one.

Reply:
Yeah, that's accidental leftovers from when I tried to book a laundry room before 1/9-17. Deleting it.